templates/example_flowsom_clustering.ipynb

```diff
@@ -104,7 +104,7 @@
     "if MIBItiff:\n",
```

Comment:
<p>This shouldn't be necessary now that you're casting to float32 within the body of the processing function, right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/352/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Are you referring to passing in <code>dtype='float32'</code> ? The reason why the images don't return the same values with <code>int16</code>  and <code>float32</code> <code>img_xr</code>  is likely in the way <code>gaussian_filter</code> works. All I can say for now is that loading <code>img_xr</code>  with <code>float32</code>  worked and loading it with <code>int16</code>  returned incorrect results and cut out a ton of pixels that shouldn't have been cut out.</p><p> </p><p>If you think it's worth researching into, I can.</p>

Reply:
ah, I see the issue now. I thought you were casting to float for the input for `gaussian_filter`, but it's actually for the output. If you cast the input to the function to `float32`, then we don't need to load the whole xarray as float32

Reply:
@ngreenwald good point, let me try that out.